package.json

```diff
@@ -16,6 +18,7 @@
     "readable-stream": "^2.3.3"
   },
   "devDependencies": {
+    "@machinomy/types-readable-stream": "git+https://github.com/machinomy/types-readable-stream.git",
```

Comment:
I don't see this module being actually used? 🤔

Reply:
I thought that was the automatic import form **WritableOptions**... Now that you mention, I will remove it -- and fix a small typo in "types" that I only saw it now.